src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs

```diff
@@ -27,12 +28,12 @@ public CheatWindow()
             Title = $"Ryujinx {Program.Version} - " + LocaleManager.Instance[LocaleKeys.CheatWindowTitle];
         }
 
-        public CheatWindow(VirtualFileSystem virtualFileSystem, string titleId, string titleName)
+        public CheatWindow(VirtualFileSystem virtualFileSystem, string titleId, string titleName, string gameBuildId)
         {
             LoadedCheats = new AvaloniaList<CheatsList>();
 
             Heading = LocaleManager.Instance.UpdateAndGetDynamicValue(LocaleKeys.CheatWindowHeading, titleName, titleId.ToUpper());
-
+            BuildId = gameBuildId;
```

Comment:
```suggestion
            BuildId = buildId;
            
```
(Just for information, this have to be changed as I suggested above.